doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_pm_optimization.rst

```diff
@@ -0,0 +1,219 @@
+It schedules the system wake-up from the system-off state in advance of the next ``GRTC`` event to compensate for the warm-boot latency of the system.
+
+Because the warm-boot latency is compensated by the System Controller, from a local CPU's perspective, the latency when restoring from the local-off state and the system-off state is expected to be the same.
+
+Latency in local domains
+========================
+
+Any local software module (such as a device driver) can anticipate events like ISRs.
+Some of these events have predictable timing, while others have unpredictable timing.
+Handling the latency of events with unpredictable timing is the same in both simple and complex systems.
+
+If handling an event with predictable timing requires restoring the state of the software module or the peripherals used by this module before the event is processed, the software module is responsible for scheduling a timer event in advance.
+This scheduled event is used to restore the state of the software module or peripherals.
+
+The power management subsystem in a local domain is responsible for scheduling a wake-up in advance to compensate for the domain's core state restoration latency from the local power state.
+The wake-up time scheduled in advance by the power management subsystem is combined with the advance time added by the software module.
+This approach ensures that the local domain and the software modules anticipating an event have sufficient time to fully restore before the event occurs, allowing the event to be handled without latency.
+
+Optimization example
+********************
+
+This example focuses on a HID device use case (for example, a mouse).
+
+A mouse has typically two main application states:
+
+* Active, when the user is moving the mouse.
+* Idle, triggered after an inactivity timeout until an input from the user is detected.
+
+The following is an example workflow for the *Active* application state:
+
+1. Sample input data from sensors.
+#. Construct a HID report.
+#. Transmit the report over the radio.
+#. Sleep until the next period (usually, less than 1 ms).
+
+Power consumption targets
+=========================
+
+The power consumption targets for this example are the following:
+
+* Sleep currents (5 V supply):
+
+  * Application core (Suspend-to-RAM): < 10 µA
+  * Radio core (Suspend-to-Idle, cache disabled): < 10 µA
+
+* Active currents (estimates at 5 V):
+
+  * Radio core in idle: ~0.25 mA
+  * App core in idle: ~0.15 mA
+  * Core wake-up every 1 ms: +0.05 mA
+  * ADC sample every 1 ms: +0.10 mA
+  * SPI transaction every 1 ms: +0.20 mA
+  * IPC message every 1 ms: +0.15 mA
+
+.. note::
+   The SPI current can be reduced to 0.03 mA by using the nRFy API instead of the default Zephyr blocking driver.
+
+Recommended Kconfig configuration
+=================================
+
+Set these Kconfig options as follows for the application running on the application core:
```

Comment:
```suggestion
For the application running on the application core, setting the following Kconfig options is recommended:
```
And similarly in line 106. This is because we are giving a recommendation. The current phrasing suggests necessity (and steps), which would require rephrasing the section and potentially more parts of the page.